src/app/public/modules/autocomplete/autocomplete-adapter.service.ts

```diff
@@ -23,12 +24,14 @@ export class SkyAutocompleteAdapterService {
     this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
   }
 
+  // Recalculate dropdown width on window resize or on skyAutocomplete focus.
   public watchDropdownWidth(elementRef: ElementRef): void {
-    Observable
-      .fromEvent(this.windowRef.getWindow(), 'resize')
-      .subscribe(() => {
-        this.setDropdownWidth(elementRef);
-      });
+    merge(
+      Observable.fromEvent(this.windowRef.getWindow(), 'resize'),
+      Observable.fromEvent(elementRef.nativeElement.querySelector('input[skyAutocomplete], textarea[skyAutocomplete]'), 'focus')
```

Comment:
Looking at this from the context of the autocomplete component, resizing the dropdown when the input is focused seems odd ("Why do I need to resize the dropdown when the input is focused?"). I know it works for the specific use case in the vertical tabs/modal component, but is there a way to repaint the dropdown that's more obvious from the autocomplete's perspective?

Is the issue related to how vertical tabs paints its hidden panels?

Reply:
The dropdown only gets redrawn once on ViewInit, and then any time a window is resized, or a user focuses on the input. After conversation, we've decided to call redraw every time a search is made. Change incoming.